image/user/db/predefined_quotes_002-devel.sql

```diff
@@ -0,0 +1,60 @@
+-- Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+-- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+INSERT OR REPLACE INTO quote_languages (lang_id, lang_name) VALUES
+    (1, 'Polski'),
+    (2, 'English'),
+    (3, 'Espanol'),
+    (4, 'Francais'),
+    (5, 'Deutsch');
```

Comment:
is adding a new language based on adding a new row with ID and string value of the language?

Reply:
Right now yes. The only thing to remember that it has to exactly the same as string of display language.

Reply:
:+1: 
so maybe a comment would be useful there?